.github/workflows/ci.yml

```diff
@@ -34,6 +34,98 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - name: Setting build env
+        run: |
+          echo "RUST_OS_NAME=${{ runner.os }}" >> $GITHUB_ENV
```

Comment:
Instead of setting `runner.os`, wouldn't it make more sense to forward `host_target`?

Reply:
I prefer not in this PR.

Reply:
I don't think we should add a complicated GHA configuration just to simplify it later. I think we do not need this concept of a "RUST_OS_NAME" at all if we can use the host target instead, so that seems clearly simpler.

Reply:
Then what if later we add linux32 build, do we need to duplicate the logic in ci.sh ?
And if we do, we have to add the HOST_TARGET env in .travis.yml or appveyor.
That's easy enough but what do you think ?

Reply:
> Then what if later we add linux32 build, do we need to duplicate the logic in ci.sh ?

Yes, we will want to do that. Which other target OSses we test should be separate for each host OS.

> And if we do, we have to add the HOST_TARGET env in .travis.yml or appveyor.

Yeah, we'll have to do that instead of adding CI_OS_NAME. Seems fine for me.